CMakeLists.txt

```diff
@@ -238,13 +238,20 @@ set(CMAKE_MODULE_PATH
 include(GNUInstallDirs)
 include(ParseVersion)
 
+set(MONGOCXX_INCLUDE_VERSION_FILE_IN_DIST OFF)
 set(MONGOCXX_CURRENT_VERSION_FILE "")
 
 if(BUILD_VERSION STREQUAL "0.0.0")
     if(EXISTS ${CMAKE_BINARY_DIR}/VERSION_CURRENT)
         file(STRINGS ${CMAKE_BINARY_DIR}/VERSION_CURRENT BUILD_VERSION)
```

Comment:
Should this `file()` be moved into the `"${CMAKE_BINARY_DIR}" STREQUAL "${PROJECT_SOURCE_DIR}/build"` branch below?

Reply:
I expect "no" to preserve the existing behavior of preferring the `${CMAKE_BINARY_DIR}/VERSION_CURRENT`. CXX-2834 has been updated to propose simplifying how the version is obtained.